### PR TITLE
Onboard the entitlements module

### PR DESCRIPTION
**Pull request by adibmbrk**
### Proposed changes in this pull request

Moving the entitlements module and related stubs from the identity-framework to the authz-xacml repo. This is done as a part of the effort to Remove XACML from the IS pack.

### CODE REVIEW

**Review by mpmadhavig — DISMISSED** [review]

**Comment by asha15** [comment]
@adibmbrk Please update the PR description.

File: components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  ~  under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.framework</groupId>
+        <artifactId>entitlement</artifactId>
+        <version>7.5.68-SNAPSHOT</version>
```
Comment (asha15) [inline components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml:26]: Please update the group ID and version. correct in other places as well

File: components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml [hunk 2]
```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (asha15) [inline components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml:3]: Update the license header.

Comment (asha15) [reply components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml:3]:
```suggestion
  ~ Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com).
```

Comment (mpmadhavig) [reply components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml:3]: I don't think we will be able to do this as this is a moving of lot of files.

Comment (adibmbrk) [reply components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml]: Done

**Pull request closed without merging**